packages/fields/src/types/Text/Implementation.js

```diff
@@ -4,10 +4,11 @@ import { KnexFieldAdapter } from '@keystonejs/adapter-knex';
 import { PrismaFieldAdapter } from '@keystonejs/adapter-prisma';
 
 export class Text extends Implementation {
-  constructor(path, { isMultiline }) {
+  constructor(path, { isMultiline, maxLength }) {
```

Comment:
I think these `Text` changes slipped in from your other PR.

Reply:
Sorry for this, I remove those code.